### PR TITLE
Login Page User Interface

### PR DESCRIPTION
**Pull request by dennisja**
##### What does this PR do?
This PR implements the login user interface

##### Description of the task to be completed.
Developing the login user interface of the application.

##### How can this be manually tested?
By running http://mrm-dev.andela.com:8080. The login UI is as seen below.
screen shot 2018-04-26 at 15 15 03

##### Relevant Pivotal Tracker Stories
- #156684727

### CODE REVIEW

File: src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from 'react';
+import '../assets/styles/login.scss';
+import darkTabletImage from '../assets/images/Dark Tablet.png';
```
Comment (JeanAbayo) [inline src/components/Login.js]: Let's use TitleCase here

Comment (shemogumbe) [reply src/components/Login.js]: I equally prefer PascalCase as opposed to camelCase

File: src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ * The login component. Currently only implements the Login  User Interface
+ * Login functionality to be Added Later
+ * @author [Dennis Jjagwe](https://github.com/dennisja)
+ */
+
+export default class Login extends Component {
+  /**
+   * Logs in a user
+   */
+  loginUser = () => {
```
Comment (shemogumbe) [inline src/components/Login.js]: Any reason for declaring this method? what value does it add at the moment?

File: __tests__/Login.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { shallow } from "enzyme";
+
+import Login from "../src/components/Login";
+
+describe("Login component", () => {
+  const shallowWrapper = shallow(<Login />);
+
+  it("renders without crashing", () => {
```
Comment (shemogumbe) [inline __tests__/Login.test.js]: The wording here is a bit scary, was it expected to crush, why not just say renders correctly? or renders as expected?

**Pull request merged**